### PR TITLE
docs: improve the grammar of the readme

### PR DESCRIPTION
**Pull request by G-Rath**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [x] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case

These are a number of small grammar issues I noticed while working on #1211 which I figured I'd do a dedicated PR for - these are not Strictly Better and there are still some improvements that could be made, but they feel like a good start :)

### Breaking Changes

This is not a breaking change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,14 +110,13 @@ For `production` mode, the `outputStyle` (old API) and `style` (new API) options
 
 Webpack provides an [advanced mechanism to resolve files](https://webpack.js.org/concepts/module-resolution/).
 
-The `sass-loader` uses Sass's custom importer feature to pass all queries to the Webpack resolving engine.
-Thus you can import your Sass modules from `node_modules`.
+The `sass-loader` uses Sass's custom importer feature to pass all queries to the Webpack resolving engine enabling you to import your Sass modules from `node_modules`.
```
Comment (evenstensberg) [inline README.md:113]: webpack is always lower case unless start of sentence

**Review by evenstensberg — COMMENTED** [review]
Ver nice, thanks a lot ✨ Left a small comment

File: README.md [hunk 2]
```diff
@@ -110,14 +110,13 @@ For `production` mode, the `outputStyle` (old API) and `style` (new API) options
...
-The `sass-loader` uses Sass's custom importer feature to pass all queries to the Webpack resolving engine.
-Thus you can import your Sass modules from `node_modules`.
+The `sass-loader` uses Sass's custom importer feature to pass all queries to the webpack resolving engine enabling you to import your Sass modules from `node_modules`.
 
 ```scss
 @import "bootstrap";
 ```
 
-Using `~` is deprecated and can be removed from your code (**we recommend it**), but we still support it for historical reasons.
+Using `~` is deprecated and should be removed from your code (**we recommend it**), but we still support it for historical reasons.
```
Comment (evenstensberg) [inline README.md:119]: should be removed, but we still recommend it? Hmm.  I think you can rephrase this to `could`be removed.

File: README.md [hunk 3]
```diff
@@ -565,10 +564,10 @@ type webpackImporter = boolean;
 
 Default: `true`
 
-Enables/Disables the default Webpack importer.
+Enables/Disables the default webpack importer.
 
-This can improve performance in some cases. Use it with caution because aliases and `@import` at-rules starting with `~` will not work.
-You can pass own `importer` to solve this (see [`importer docs`](https://github.com/sass/node-sass#importer--v200---experimental)).
+This can improve performance in some cases though use it with caution because aliases and `@import` at-rules starting with `~` will not work.
```
Comment (evenstensberg) [inline README.md:569]: remove `though` and use a comma instead

**Review by evenstensberg — COMMENTED** [review]
Some more tweaks and I think we are good to approve this!

File: README.md (hunk 2 above)
Comment (G-Rath) [reply README.md:119]: it's saying that we recommend _removing it_, hence why the use of "should" here - this sentence (like a number of others) could definitely be improved further, but I'm wanting to avoid taking on too much in the one PR for bandwidth reasons and I think it's worth landing the low hanging wins before attempting any bigger changes :)

File: README.md (hunk 3 above)
Comment (G-Rath) [reply README.md:569]: the "though" is required to join the two sentences - it's not grammatically correct to omit it, but agree about adding the comma to make it less awkward.

File: README.md (hunk 2 above)
Comment (G-Rath) [reply README.md:119]: tbh frankly I think it's probably best short-term at least to just remove the "recommendation" - the fact that it's deprecated already signals that it's recommended to use alternatives, and this just confuses things further 🤷

**Review by evenstensberg — APPROVED** [review]
Good job! Thx & congrats on your first contribution!🌟

**Comment by G-Rath** [comment]
Looks like CI is failing due to https://github.com/sass/node-sass/issues/3420 and that it might not get fixed in favor of officially marked as [unmaintained](https://github.com/sass/node-sass/issues/3424) 🤔

**Comment by evenstensberg** [comment]
This is just a docs change, so we'll merge anyway

**Comment by G-Rath** [comment]
cool, thanks - I'll see if this can be worked around in the meantime by just installing a different version of Python in CI, which'll be easier now that I'm no longer a first-time contributor 🎉

**Comment by evenstensberg** [comment]
Awesome,good stuff 🙏

**Comment by G-Rath** [comment]
Have opened #1215 to fix the `node-sass` issue
